### ISSUE TITLE
request: function to re-run tblastn to rotate to start_gene after manual finnishing

### ISSUE THREAD

**Issue by thsyd** [rrwick/Unicycler#151]
Would it be possible to include an option to re-run the tblastn and rotate to starting positition? 
This is only run on circular contigs, and if you have completed a circular contig by manual finishing, it would be great to just make use of the function allready included in Unicycler.
Perhaps provide as a seperate tool, as is done with e.g. https://github.com/rrwick/Unicycler/blob/master/docs/unicycler-polish.md

**Comment by alexweisberg** [comment]
I agree, this would be an excellent feature, as well as to be able to change which starting gene to blast by in case you change your mind.

**Comment by thsyd** [comment]
I guess we can use the fixstart task in Circlator. https://github.com/sanger-pathogens/circlator/wiki/Task%3A-fixstart
